src/SFA.DAS.EmployerFinance.Web/Views/Transfers/FinancialBreakdown.cshtml

```diff
@@ -85,7 +85,7 @@
         <details>
             <summary><span class="summary">How are my committed funds calculated?</span></summary>
             <div class="panel panel-border-narrow">
-                <p>Content</p>
+                <p>Your committed funds are calculated by subtracting your <b>total estimated spend</b> from your <b>@Model.Data.TotalAvailableTransferAllowance.ToString("C0", culture) total available transfer allowance</b>.</p>
```

Comment:
It is conventional to use `<strong>` over `<b>`

Reply:
Strong used now instead of b